src/main/plugins/org.dita.base/build_init.xml

```diff
@@ -45,6 +45,27 @@ See the accompanying LICENSE file for applicable license.
     </sequential>
   </macrodef>
 
+  <macrodef name="preprocess-skip-init">
```

Comment:
This may not be the best name, because it can be used for transtype specific targets too. So maybe `init-skip-property`?